schemas/common/event/rejected.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/xdm/common/event/rejected",
+  "type": "object",
+  "title": "Rejected Event",
+  "description": "A `rejected event` follows semantics that are exactly equivalent to the semantics described by the [`reject activity`](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-reject), except for one difference. The `reject activity` can be used in imperative contexts, however, the `rejected event` can only be used in passive contexts.",
```

Comment:
I don't get it. Is there a way to re-phrase the description? If it is getting longer, a separate file might be useful.

What are examples of objects that can be rejected? I can see the Adobe Stock example, but the description should list additional examples.

Reply:
I have added a better description. Please suggest any other changes, if needed.